extensions/lima/src/extension.ts

```diff
@@ -68,12 +69,16 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
   const socketPath = path.resolve(limaHome, instanceName + '/sock/' + engineType + '.sock');
   const configPath = path.resolve(limaHome, instanceName + '/copied-from-guest/kubeconfig.yaml');
 
+  const installedLima = await getLimaInstallation();
+  const version: string | undefined = installedLima?.version;
+
   let provider;
   if (fs.existsSync(socketPath) || fs.existsSync(configPath)) {
     provider = extensionApi.provider.createProvider({
       name: 'Lima',
       id: 'lima',
       status: 'unknown',
+      version,
```

Comment:
it might be better to update the version when you've the result and not hold the activate method

activate method should be a 'fast call', (as we wait the ending of the activate method on all extensions before showing the UI)

so probably, in activate you register the provider, then asynchronously you grab the version and call provider.updateVersion

Reply:
It's a local binary, but sure.